editable-templates/mvp-reference-pages/nested-columns-component.html

```diff
@@ -0,0 +1,62 @@
+---
+title: Nested-columns component
+hide_breadcrumb: false
+date_modified: 2024-10-31
+layout: no-container
+---
+
+<div class="">
+	<div class="container">
+		<div class="">
+			<div class="row">
+				<div class="title col-lg-12 col-xs-12">
+					<h1 property="name" id="wb-cont" dir="ltr"> Nested columns component
+					</h1>
+				</div>
+				<div class="text col-lg-12">
+					<div id="text-54cd5cba73" class="cmp-text">
+						Nested columns component added inside a group component
+					</div>
+				</div>
+				<div class="col-lg-12 col-xs-12">
+					<div class="cmp-container">
+						<div class="nested-columns mwscolumns">
+							<div class="row">
+								<div class="col-md-4">
+									<div class="text section">
+										<div id="text-83c0f8de7a" class="cmp-text">
+											<div class="text-justify">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque
```

Comment:
Note / Optimization: We should consider to hide the "text-justify" option or make it more difficult to use. It is known that style are challenging for user from an accessibility perspective.